src/components/SubCategory.js

```diff
@@ -24,36 +23,40 @@ const StyledSubCategory = styled.div`
     flex-wrap: wrap;
     margin-left: 10px;
   }
-`;
+`
 
 const subCategoryInfo = {
-    id: 'unique',
-    title: 'Books/Movies/Music',
-  }
-      
-const subCategoryEntries = [
-  {title: "Mistborn", amount: 9.99},
-  {title: "Name of the Wind", amount: 24.95},
-  {title: "The Dark Knight", amount: 8.99},
-  {title: "The Marvelous Mrs. Maisel", amount: 4.92},
-  {title: "Mumford & Sons Vinyl", amount: 15.65}
-];
+  id: 'unique',
+  title: 'Books/Movies/Music',
+}
 
-const subCategoryTotal = subCategoryEntries.reduce((nextEntry, totalEntry) => 
-  nextEntry + totalEntry.amount, 0);
+const subCategoryEntries = [
+  { title: 'Mistborn', amount: 9.99 },
+  { title: 'Name of the Wind', amount: 24.95 },
+  { title: 'The Dark Knight', amount: 8.99 },
+  { title: 'The Marvelous Mrs. Maisel', amount: 4.92 },
+  { title: 'Mumford & Sons Vinyl', amount: 15.65 },
+]
+
+export const subCategoryTotal = subCategoryEntries.reduce(
+  (nextEntry, totalEntry) => nextEntry + totalEntry.amount,
+  0
+)
 
 export default function SubCategory(props) {
-    return (
-     <StyledSubCategory>
-      <div className='SubCategory-header'>
+  return (
+    <StyledSubCategory>
+      <div className="SubCategory-header">
         <span>{subCategoryInfo.title}</span>
-        <span className='SubCategory-total'>${subCategoryTotal}</span>
+        <span className="SubCategory-total">
+          {numToCurrency(subCategoryTotal)}
+        </span>
       </div>
-      <ul className='SubCategory-entries'>
-        {subCategoryEntries.map(entryItem => 
-           <Entry {...entryItem} />
-        )}
+      <ul className="SubCategory-entries">
+        {subCategoryEntries.map(entryItem => (
```

Comment:
es-lint should be yelling at you here cause you don't have a key, ping me when you're working and we'll go over what it is